src/matter/cluster/server/LevelControlServer.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { LevelControlCluster } from "../LevelControlCluster";
+import { ClusterServerHandlers } from "./ClusterServer";
+
+/*
+TODO:  not clear about how parameters get passed and returned. And there is logic
+that needs to be added for changing other clusters from the WithOnOff commands and to
+set attributes other than currentLevel
+
+The moveToLevel commands have been tested in Google Home and in Alexa
+*/
+
+export const LevelControlClusterHandler: () => ClusterServerHandlers<typeof LevelControlCluster> = () => ({
+    moveToLevel: async ({ request:{level, transitionTime, optionsMask, optionsOverride}, attributes: { currentLevel } }) => { 
+      currentLevel.set(level);
```

Comment:
Add a todo to implement the optionmask handling as described in 1.6.6.1.1 (or you implent it ... it is a bit of bit fun :-)

Reply:
Will get this PR submitted then will take a look at the bitmap. Although I'd be more comfortable doing bit-twiddling in C rather than Typescript ;-)